src/components/JLCPCBImportDialog.tsx

```diff
@@ -121,32 +119,53 @@ export function JLCPCBImportDialog({
             className="mt-3"
             placeholder="Enter JLCPCB part number (e.g., C46749)"
             value={partNumber}
-            onChange={(e) => setPartNumber(e.target.value)}
+            onChange={(e) => {
+              setPartNumber(e.target.value)
+              setError(null)
+              setAlreadyImportedPackageId(null)
+            }}
           />
-          {error && <p className="bg-red-100 p-2 mt-2 pre-wrap">{error}</p>}
-          {error && (
+          {error && !alreadyImportedPackageId && (
+            <p className="bg-red-100 p-2 mt-2 pre-wrap">{error}</p>
+          )}
+
+          {error && !alreadyImportedPackageId && (
             <div className="flex justify-end mt-2">
               <Button
                 variant="default"
                 onClick={() => {
                   const issueTitle = `[${partNumber}] Failed to import from JLCPCB`
                   const issueBody = `I tried to import the part number ${partNumber} from JLCPCB, but it failed. Here's the error I got:\n\`\`\`\n${error}\n\`\`\`\n\nCould be an issue in \`fetchEasyEDAComponent\` or \`convertRawEasyEdaToTs\``
                   const issueLabels = "snippets,good first issue"
-                  const url = `https://github.com/tscircuit/easyeda-converter/issues/new?title=${encodeURIComponent(issueTitle)}&body=${encodeURIComponent(issueBody)}&labels=${encodeURIComponent(issueLabels)}`
-
-                  // Open the issue in a new tab
+                  const url = `https://github.com/tscircuit/easyeda-converter/issues/new?title=${encodeURIComponent(
+                    issueTitle,
+                  )}&body=${encodeURIComponent(
+                    issueBody,
+                  )}&labels=${encodeURIComponent(issueLabels)}`
                   window.open(url, "_blank")
                 }}
               >
-                File Issue on Github (prefilled)
+                File Issue on GitHub (prefilled)
               </Button>
             </div>
           )}
+
+          {alreadyImportedPackageId && (
+            <p className="mt-2 text-sm text-green-600">
+              This part number has already been imported to your profile.{" "}
+              <PrefetchPageLink
+                className="text-blue-500 hover:underline"
+                href={`/editor?package_id=${alreadyImportedPackageId}`}
```

Comment:
feels like it would make more sense to not link to the editor, but to link to the package page, `/author/packageName`